orttraining/orttraining/core/framework/ortmodule_graph_builder.cc

```diff
@@ -84,7 +85,7 @@ Status OrtModuleGraphBuilder::Build(const std::vector<std::vector<int64_t>>* inp
 
   // Optimize the inference graph and build the gradient graph.
```

Comment:
nit: can you change the comment to Optimize the forward graph instead of inference graph. Thanks!